src/web/components/elements/WitnessVideoBlockComponent.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { space } from '@guardian/src-foundations';
+import { headline, body } from '@guardian/src-foundations/typography';
+
+import { pillarPalette } from '@root/src/lib/pillars';
+import { WitnessWrapper } from '@frontend/web/components/WitnessWrapper';
+
+type Props = {
+	title: string;
+	description: string;
+	authorName: string;
+	width: number;
+	height: number;
+	youtubeUrl: string;
+	dateCreated: string;
+	pillar: Theme;
+};
+
+const captionStyles = css`
+	margin-top: ${space[3]}px;
+`;
+
+const titleStyles = (pillar: Theme) => css`
+	margin-bottom: ${space[2]}px;
+	color: ${pillarPalette[pillar].main};
+	${headline.xxxsmall()}
+`;
+
+export const WitnessVideoBlockComponent = ({
+	title,
+	description,
+	authorName,
+	width,
+	height,
+	youtubeUrl,
```

Comment:
If we're not using an atom here and if we have zero control over what this url is, isn't the most we can do just use it? Rather than trying to parse it?